examples/zookeeper/src/test/java/com/example/ZookeeperContainerTest.java

```diff
@@ -0,0 +1,39 @@
+package com.example;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.RetryOneTime;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+
+import java.nio.charset.StandardCharsets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ZookeeperContainerTest {
+
+    private static final int ZOOKEEPER_PORT = 2181;
+
+    @Test
+    public void test() throws Exception {
+        String path = "/messages/zk-tc";
+        String content = "Running Zookeeper with Testcontainers";
+        try(GenericContainer<?> zookeeper = new GenericContainer<>("zookeeper:3.8.0")
+            .withExposedPorts(ZOOKEEPER_PORT)) {
```

Comment:
Do we apply spotless to our examples? It seems to miss a line break after `try(`

Reply:
See https://github.com/testcontainers/testcontainers-java/pull/6061

Reply:
So we merge after #6061?